docs/msi_v2/slc_revocation_spec.md

```diff
@@ -1,46 +1,46 @@
-# Short-Lived Credential (SLC) Revocation Specification
+# Certificate Revocation Specification
 
 ## Overview
 
-This document outlines the design and implementation details for short-lived credential (SLC) revocation in MSI V2 scenarios. 
+This document outlines the design and implementation details for **certificate revocation** in MSI V2 scenarios.
```

Comment:
Should we have 1 document that discusses all credential revocation ? (cert and token)

Reply:
yes